presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -84,6 +84,13 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Tables with Unsupported Columns
+-------------------------------
+
+When you query a PostgreSQL table via the Presto connector, and the table either has no supported columns or contains
+only unsupported data types, Presto will return an error. The error message will explicitly state the problem with the
```

Comment:
```suggestion
only unsupported data types, Presto will return an error. The error message will be similar to the following example:
```
You described the error message in #21422 . Consider adding the error message here for the user to read.